index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <title>Microverse Portfolio project</title>
+  </head>
+  <body>
+    <header>
+      <div class="Logo">
+        <h4><a href="#"></a>Solomon Afutu</h4>
```

Comment:
- [ ] If you notice, most of the websites have a logo, such that when the user clicks, the page is redirected to the home page ☺️ . This means the logo is always `a` tag. Therefore, I suggest that you enclose the logo in the link.
- [ ] You should also consider slightly increasing the size of the logo and also try to increase its weight to make it bold.